legal-api/tests/unit/services/filings/validations/change_of_director/test_validation_effective_date.py

```diff
@@ -56,130 +50,153 @@ def test_effective_date_sanity_check(session):
-    filing_json['filing']['header']['effectiveDate'] = (now - datedelta.MONTH).isoformat()
-    filing_json['filing']['changeOfDirectors'] = copy.deepcopy(CHANGE_OF_DIRECTORS)
-
-    business = Business(identifier=identifier,
-                        founding_date=now - datedelta.datedelta(years=4)
-                        )
-    business.save()
-
-    # create a COD
-    factory_completed_filing(business=business,
-                             data_dict=filing_json,
-                             filing_date=(now - datedelta.MONTH))
-
-    # move the COD to now
-    filing_json['filing']['header']['effectiveDate'] = now.isoformat()
+    # assign 'now' with non-zero hour so Founding Date is mid-day
+    now = datetime(2001, 8, 5, 12, 0, 0, 0, tzinfo=timezone.utc)
+    business, filing = common_setup(identifier, now)
 
     with freeze_time(now):
-        err = validate_effective_date(business, filing_json)
+        err = validate_effective_date(business, filing)
     assert not err
 
 
 def test_validate_effective_date_not_in_future(session):
     """Assert that the effective date of change cannot be in the future."""
     # setup
     identifier = 'CP1234567'
-    now = datetime(2001, 8, 5, 0, 0, 0, 0, tzinfo=timezone.utc)
-
+    # assign 'now' with non-zero hour so Founding Date is mid-day
+    now = datetime(2001, 8, 5, 12, 0, 0, 0, tzinfo=timezone.utc)
     business, filing = common_setup(identifier, now)
 
-    # The effective date of change cannot be in the future
+    # The effective date _cannot_ be in the future.
+    effective_date = datetime(2001, 8, 6, 0, 0, 0, 0, tzinfo=timezone.utc)
+    filing['filing']['header']['effectiveDate'] = effective_date.isoformat()
+    with freeze_time(now):
+        err = validate_effective_date(business, filing)
+    assert err
+
+    # The effective date _can_ be today.
+    effective_date = datetime(2001, 8, 5, 0, 0, 0, 0, tzinfo=timezone.utc)
+    filing['filing']['header']['effectiveDate'] = effective_date.isoformat()
     with freeze_time(now):
         err = validate_effective_date(business, filing)
     assert not err
 
+    # The effective date _can_ be in the past.
+    effective_date = datetime(2001, 8, 4, 0, 0, 0, 0, tzinfo=timezone.utc)
+    filing['filing']['header']['effectiveDate'] = effective_date.isoformat()
+    with freeze_time(now):
+        err = validate_effective_date(business, filing)
+    assert not err
 
-def test_validate_effective_date_not_before_founding(session):
-    """Assert the filing is not before the business was founded.
 
-    Rules:
-        - The effective date cannot be a date prior to their Incorporation Date
-    """
+def test_validate_effective_date_not_before_founding(session):
+    """Assert that the effective date cannot be a date prior to their Incorporation Date."""
     # setup
     identifier = 'CP1234567'
-    now = datetime(2001, 8, 5, 0, 0, 0, 0, tzinfo=timezone.utc)
-
+    # assign 'now' with non-zero hour so Founding Date is mid-day
```

Comment:
this is not mid-day for the legal timezone.

Reply:
You're right. It's just not "0 h". I can reword.

Reply:
I may have confused myself about what "time" things happen. All date comparisons should be in local time right?
I will review these and then re-submit for PR.

Reply:
Updated in next commit.

Reply:
they can all be local, utc, or mixed (harder for humans to reason). Date comparisons using UTC are preferable.